main/explicit_bzero.c

```diff
@@ -30,15 +30,19 @@
 
 #include <string.h>
 
-__attribute__((weak)) void
-__explicit_bzero_hook(void *dst, size_t siz)
-{
-}
-
 PHPAPI void php_explicit_bzero(void *dst, size_t siz)
 {
+#ifndef __GNUC__
```

Comment:
Looks like this condition got inverted.

Reply:
ended up with bad merges ... recommitting...